src/java.base/share/classes/java/lang/ClassValue.java

```diff
@@ -347,16 +369,11 @@ boolean isLive() {
             Version<T> v = version();
             if (v == null)  return false;
             if (v.isLive())  return true;
```

Comment:
```suggestion
            if (v == null)  return false;
            if (v.isLive()) return true;
```
Remove a space to align the code